rpcs3/Emu/Io/Skylander.cpp

```diff
@@ -272,7 +272,13 @@ void usb_device_skylander::control_transfer(u8 bmRequestType, u8 bRequest, u16 w
 				// Shutdowns the portal
 				ensure(buf_size == 2 || buf_size == 32);
 				q_result = {
-				    0x52, 0x02, 0x0A, 0x03, 0x02, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
+					0x53, 0x00, 0x00, 0x00, 0x00, 0xc4, 0x01, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00};
```

Comment:
0x53 packets are sent when there is a query but no pending data to get status of figures. Also you're using static values for the packet number which is definitely incorrect, see https://github.com/RPCS3/rpcs3/blob/769c7bfb9180d85a18905ded73ae67b8807ccccd/rpcs3/Emu/Io/Skylander.cpp#L74 .

Reply:
Thanks for pointing that out.   Initially it appeared the status packets were required but after further testing it appears it was just a timing issue reading the portal.   The only real change needed is the version info in byte 2/3 from 0A03 to 1800.

That being said, there's a few additional query types I'm tracking down to see if they require any real response.  It doesn't appear so but I'd rather be sure.

I'll update the PR tonight once I find those.

Reply:
i dont understand this at all

Reply:
please tell me where I need to put that code if u can please beacuse my traps arent working in game when I use the emulator portal